flink-dist/pom.xml

```diff
@@ -460,50 +460,6 @@ under the License.
 			</dependencies>
 		</profile>
 
-		<profile>
-			<!-- Copies that shaded Hadoop uber jar to the dist folder. -->
-			<id>include-hadoop</id>
```

Comment:
still referenced in the azure files and `run-nightly-tests.sh`

Reply:
You are right. The e2e tests will always have access to at least Hadoop 2.4.1, and we can remove the "hadoopfree" nightly profile

Reply:
Addressed in https://github.com/rmetzger/flink/commit/eda372b26dc5ea82b03f0ada8d16bc85afc61123